File: Data Structures & Algorithms/daily-temperatures/submission-0.java
class Solution {
    public int[] dailyTemperatures(int[] temperatures) {
        int[] returning = new int[temperatures.length];
        for (int i = 0; i < temperatures.length; i++) {
            int count = 0;
            for (int k = i + 1; k < temperatures.length; k++) {
                if (temperatures[k] > temperatures[i]) {
                    returning[i] = k - i;
                    count++;
                    break;
                } 
            }
            if (count == 0) {
                returning[i] = 0;
            }
        }
        return returning;
    }
}

File: Data Structures & Algorithms/count-primes/submission-0.cpp
class Solution {
public:
    int countPrimes(int n) {
        if (n < 3) return 0;

        vector<bool> isPrime(n, false);
        int count = n / 2;

        for (int i = 3; i * i < n; i += 2) {
            if (!isPrime[i]) {
                for (int j = i * i; j < n; j += 2 * i) {
                    if (!isPrime[j]) {
                        isPrime[j] = true;
                        count--;
                    }
                }
            }
        }

        return count;
    }
};